modules/110-istio/templates/cni/configmap.yaml

```
@@ -3,7 +3,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: istio-cni-config
+  name: d8-istio-cni-config
```

```suggestion
  name: cni-config
```
it is in namespace d8-istio.